rdr_service/offline/sync_consent_files.py

```diff
@@ -72,6 +73,7 @@ def copy_file(source, destination, participant_id, dry_run=True, zip_files=False
 
 def archive_and_upload_consents(dry_run=True):
     logging.info("zipping and uploading consent files...")
+    storage_provider = GoogleCloudStorageProvider()
     for bucket_dir in _directories_in(TEMP_CONSENTS_PATH):
         for org_dir in _directories_in(bucket_dir):
```

Comment:
I think you should use the `get_storage_provider()` function in `rdr_service/storage.py` here, as this can correctly get either the local or cloud storage provider depending on where the code is being run.

Reply:
I may be misunderstanding, but I don't think I would want it to use the LocalStorageProvider whether we're running the tool on our machines or if it's running in the cloud as part of the cron job. In either case I want to access the GoogleCloudStorageProvider's implementation of `upload_from_file`.

Reply:
Ok.  I just wanted to make sure you were aware of the `get_storage_provider()` function.